utility/types/proto/actor.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+package utility;
+
+option go_package = "github.com/pokt-network/pocket/utility/types";
+
+enum Pool_Names {
+  DAO = 0;
+  FeeCollector = 1;
+  AppStakePool = 2;
+  ValidatorStakePool = 3;
+  ServiceNodeStakePool = 4;
+  FishermanStakePool = 5;
+}
+
+// DISCUSS: similar code exists in the persistence module.
+// Talk about appropriate balance of shared code / encapsulation
+enum ActorType {
+  App = 0;
+  Node = 1;
+  Fish = 2;
+  Val = 3;
```

Comment:
We use `FishermanStakePool` and `ValidatorStakePool` above, so I think we should use the full name here as well. i.e.:

1. `ServiceNode`
2. `Fisherman`
3. `Validator`